app/src/main/java/net/gsantner/markor/format/FormatRegistry.java

```diff
@@ -74,6 +75,32 @@ public class FormatRegistry {
     public final static OrgmodeTextConverter CONVERTER_ORGMODE = new OrgmodeTextConverter();
 
 
+
+    public static class Format {
+        public final @StringRes int format, name;
+        public final String ext;
```

Comment:
Suggestion please make ext / it's handling a regex instead

Reply:
These are just meant to be a source of truth, not used for matching etc. i.e. this says ' The canonical extension for markdown is ".md" '

Format and FORMATS is not meant to match or detect formats - that is left to the `isFileOutOfThisFormat` as before.